brands/tourism/hotel.json

```diff
@@ -467,6 +537,15 @@
       "tourism": "hotel"
     }
   },
+  "tourism/hotel|Marriott Executive Apartments": {
+    "tags": {
+      "brand": "Marriott Executive Apartments",
+      "brand:wikidata": "Q1053170",
+      "brand:wikipedia": "en:Marriott International",
```

Comment:
If Marriott Executive Apartments is a variation of Marriott’s flagship hotel brand, then we could tag it with [Q3918608](https://en.wikipedia.org/wiki/Marriott_Hotels_%26_Resorts). Otherwise, let’s create a separate Wikidata item for it rather than link to the parent company’s Wikidata item and Wikipedia article. `brand` and `operator` tags aren’t really designed for expressing corporate hierarchies.

Reply:
I'll create an entry for this one as well

Reply:
Fixed